### PR TITLE
Align credentials stores - part 30

### PR DESCRIPTION
**Pull request by maimorag**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Relates:[ link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-5274)

## Description
Change the type of credentials, from type 4 to type 9.

## Minimum version of Cortex XSOAR
- [x] 6.5.0

## Does it break backward compatibility?
  - [x] No

### CODE REVIEW

File: Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/AzureLogAnalytics.py [hunk 1]
```diff
@@ -348,10 +353,10 @@ def main():
         client = Client(
             self_deployed=self_deployed,
             auth_and_token_url=auth_and_token_url,  # client_id or auth_id
-            refresh_token=params.get('refresh_token'),  # tenant_id or token
+            refresh_token=refresh_token,  # tenant_id or token
             enc_key=enc_key,  # client_secret or enc_key
             redirect_uri=params.get('redirect_uri', ''),
-            auth_code=params.get('auth_code') if not client_credentials else '',
+            auth_code=auth_code,
```
Comment (Shellyber) [inline Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/AzureLogAnalytics.py:359]: Add back the comment you removed

File: Packs/XSOARmirroring/Integrations/XSOARmirroring/XSOARmirroring.py [hunk 2]
```diff
@@ -757,7 +757,11 @@ def update_remote_system_command(client: Client, args: Dict[str, Any], mirror_ta
 
 
 def main() -> None:
-    api_key = demisto.params().get('apikey')
+    # If you want to mirror from server X to self-service Y, create an API
```
Comment (Shellyber) [inline Packs/XSOARmirroring/Integrations/XSOARmirroring/XSOARmirroring.py:760]: Remove the comment

**Review by Shellyber — COMMENTED** [review]
Can't comment there for some reason, but check in the test_connection function regarding the auth_code condition. You should change here as well, no?

File: Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/AzureLogAnalytics.py (hunk 1 above)
Comment (maimorag) [reply Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/AzureLogAnalytics.py]: done

Comment (maimorag) [reply Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/AzureLogAnalytics.py]: updating test_connection

File: Packs/AzureLogAnalytics/ReleaseNotes/1_1_13.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Azure Log Analytics
+- Added the *Token / Tenant ID*, *Authorization code* and *Certificate Thumbprint* integration parameters to support credentials fetching object.
```
Comment (Shellyber) [inline Packs/AzureLogAnalytics/ReleaseNotes/1_1_13.md:5]: Change the release notes according to our standard (what has changed as list)

**Review by Shellyber — APPROVED** [review]

**Pull request merged**